envoy-control-core/src/main/kotlin/pl/allegro/tech/servicemesh/envoycontrol/groups/NodeMetadata.kt

```diff
@@ -182,16 +182,25 @@ fun Value?.toOutgoing(properties: SnapshotProperties): Outgoing {
     val defaultSettingsFromProperties = defaultDependencySettings(properties)
     val defaultSettings = this.toSettings(defaultSettingsFromProperties)
     val allServicesDefaultSettings = allServicesDependencies?.value.toSettings(defaultSettings)
-    val services = rawDependencies.filter { it.service != null && it.service != allServiceDependenciesIdentifier }
+    val services = rawDependencies
+        .filter { it.service != null && it.service != allServiceDependenciesIdentifier }
         .map {
             ServiceDependency(
                 service = it.service.orEmpty(),
                 settings = it.value.toSettings(allServicesDefaultSettings)
             )
+        } +
+        properties.defaultDependencies.services.map {
```

Comment:
I don't like `+` operator, but its only my opinion, so it can stay. But please make it more readable, cause right now it looks like some random stuff not connected to each other